src/components/KaotoToolbar.tsx

```diff
@@ -169,7 +170,10 @@ export const KaotoToolbar = ({ toggleCatalog, toggleCodeEditor }: IKaotoToolbar)
     <DropdownItem
       key="settings"
       data-testid={'kaotoToolbar-kebab__settings'}
-      onClick={() => setExpanded({ ...expanded, settingsModal: !expanded.settingsModal })}
+      onClick={() => {
```

Comment:
Did you forget to remove this extra space? Just this is causing the code coverage to go down, which is so funny. 😂